x-pack/plugin/core/src/main/resources/logs-template.json

```diff
@@ -4,9 +4,11 @@
   "data_stream": {},
   "composed_of": [
     "logs-mappings",
-    "data-streams-mappings",
+    "ecs-dynamic-mappings",
```

Comment:
What is the convention you are following for the naming? ;-) https://github.com/elastic/elasticsearch/issues/96267